Kconfig.zephyr

```diff
@@ -1048,3 +1048,18 @@ config BOOTLOADER_BOSSA_ADAFRUIT_UF2
 endchoice
 
 endmenu
+
+menu "Compatibility"
+
+config LEGACY_GENERATED_INCLUDE_PATH
+	bool "Legacy include path for generated headers"
+	default y
```

Comment:
Shouldn't this be disabled by default? Otherwise we risk introducing new include statements using the old path.

Reply:
Removed `default y` and added `select DEPRECATED`